account_move_template/wizard/account_move_template_run.py

```diff
@@ -0,0 +1,301 @@
+# Copyright 2015-2019 See manifest
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from ast import literal_eval
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+
+class AccountMoveTemplateRun(models.TransientModel):
+    _name = "account.move.template.run"
+    _description = "Wizard to generate move from template"
+
+    template_id = fields.Many2one("account.move.template", required=True)
+    company_id = fields.Many2one(
+        "res.company",
+        required=True,
+        readonly=True,
+        default=lambda self: self.env.company,
+    )
+    partner_id = fields.Many2one(
+        "res.partner",
+        "Override Partner",
+        domain=["|", ("parent_id", "=", False), ("is_company", "=", True)],
+    )
+    date = fields.Date(required=True, default=fields.Date.context_today)
+    journal_id = fields.Many2one("account.journal", string="Journal", readonly=True)
+    ref = fields.Char(string="Reference")
+    line_ids = fields.One2many(
+        "account.move.template.line.run", "wizard_id", string="Lines"
+    )
+    state = fields.Selection(
+        [("select_template", "Select Template"), ("set_lines", "Set Lines")],
+        readonly=True,
+        default="select_template",
+    )
+    overwrite = fields.Text(
+        help="""
+Valid dictionary to overwrite template lines:
+{'L1': {'partner_id': 1, 'amount': 100, 'name': 'some label'},
+ 'L2': {'partner_id': 2, 'amount': 200, 'name': 'some label 2'}, }
+        """
+    )
+
+    def _prepare_wizard_line(self, tmpl_line):
+        vals = {
+            "wizard_id": self.id,
+            "sequence": tmpl_line.sequence,
+            "name": tmpl_line.name,
+            "amount": 0.0,
+            "account_id": tmpl_line.account_id.id,
+            "partner_id": tmpl_line.partner_id.id or False,
+            "move_line_type": tmpl_line.move_line_type,
+            "tax_line_id": tmpl_line.tax_line_id.id,
+            "tax_ids": [(6, 0, tmpl_line.tax_ids.ids)],
```

Comment:
Migrate to the new commands syntax:

```suggestion
            "tax_ids": [Command.SET(tmpl_line.tax_ids.ids)],
```

Same in all similar appearances.

Also same for `[(4, ...)]`, which is replaced by `Comand.LINK`